packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte

```diff
@@ -13,7 +13,19 @@
   title="Components / Structure / VtmnCard"
   component={VtmnCard}
   {parameters}
-  {argTypes}
+  argTypes={{
+    ...argTypes,
+    headingLevel: {
+      type: { name: 'string', required: false },
+      description: 'Heading level of the card title',
+      defaultValue: undefined,
+      control: {
+        type: 'select',
+        options: [undefined, 1, 2, 3, 4, 5, 6],
+        default: undefined,
```

Comment:
do we want an undefined heading level ? 2 or 3 could be a valid default value don't you think ?

Reply:
We have to keep a value undefined not to break the current card that have a 'span'.

Reply:
I don't understand. If there is a default value then the prop is optional then existing card without the prop and with the span would not break, no ? Am I missing something ?

Reply:
Yes, you are right. It shouldn't impact current Cards